public/javascripts/SVLabel/src/SVLabel/modal/ModalMissionComplete.js

```diff
@@ -268,11 +268,12 @@ function ModalMissionComplete (svl, missionContainer, missionModel, taskContaine
     };
 
     this.update = function (mission, neighborhood) {
-        // Update the horizontal bar chart to show how much distance the user has audited
-        var unit = {units: 'miles'};
+        // Update the horizontal bar chart to show how much distance the user has audited.
```

Comment:
"how much distance the user has audited" -> "how much the user has audited" or "the distance the user has audited"